### ISSUE TITLE
Issues with zip files

### ISSUE THREAD

**Issue by jdm** [jdm/mzxscreen#4]
https://www.digitalmzx.com/show.php?id=2496
```
     Running `target/debug/populate download.zip output.json`
Fetching https://www.digitalmzx.com/search.php?browse=ALL&page=27
Fetching https://www.digitalmzx.com/show.php?id=2496
Fetching https://www.digitalmzx.com/download/2496/2cb1088a453e9a49b73d8ec43274b4aba3f7c81a7f4abc2b5431be23900033ec/
error: open zip entry PUN1.MZX failed
thread 'main' panicked at 'assertion failed: `(left == right)`
  left: `7`,
 right: `0`', populate/src/main.rs:104:5
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

**Comment by jdm** [comment]
https://www.digitalmzx.com/show.php?id=944

**Comment by jdm** [comment]
Fixed by b1ff894c085108313c7b6114f4be172e053dec62.
